decidim-core/spec/lib/reminder_registry_spec.rb

```diff
@@ -37,8 +37,10 @@ def register_reminder(name, generator_class_name = "Decidim::DummyGenerator")
       end
     end
   end
+  # rubocop:disable Lint/EmptyClass
```

Comment:
You could also avoid this by extending the dummy classes from some other class, e.g.

```ruby
class DummyGenerator < Object; end
```